test/ArcxAnalyticsSdk.test.ts

```diff
@@ -44,72 +47,81 @@ describe('(unit) ArcxAnalyticsSdk', () => {
 
   describe('#init', () => {
     it('posts identify', async () => {
-      await ArcxAnalyticsSdk.init('', TEST_CONFIG)
+      await ArcxAnalyticsSdk.init('', ALL_FALSE_CONFIG)
       expect(postRequestStub.calledOnceWith(PROD_URL_BACKEND, '', '/identify')).to.be.true
     })
 
     it('sets the current URL in the session storage when tracking pages', async () => {
       expect(sessionStorage.getItem(CURRENT_URL_KEY)).to.be.null
-      expect(location.href).to.eq('http://localhost:3000/')
+      reconfigureJsdom({
+        url: 'https://arcx.money',
+      })
+      expect(window.location.href).to.eq('https://arcx.money/')
 
       await ArcxAnalyticsSdk.init('', { trackPages: true })
 
-      expect(sessionStorage.getItem(CURRENT_URL_KEY)).to.eq('http://localhost:3000/')
+      expect(sessionStorage.getItem(CURRENT_URL_KEY)).to.eq('https://arcx.money/')
     })
 
-    it('makes an initial REFERRER call', async () => {
+    it('makes an initial FIRST_PAGE_VISIT call url, utm and referrer if using the default config', async () => {
+      const url = 'https://example.com/?utm_source=google&utm_medium=cpc&utm_campaign=brand'
+
       globalJsdom('', {
+        url,
         referrer: 'https://arcx.money',
       })
 
-      expect(document.referrer).to.eq('https://arcx.money/')
-
-      await ArcxAnalyticsSdk.init('', {
-        trackReferrer: true,
-        trackPages: false,
-        cacheIdentity: false,
-      })
+      await ArcxAnalyticsSdk.init('', { cacheIdentity: false })
       expect(postRequestStub.getCall(0).calledWith(PROD_URL_BACKEND, '', '/identify')).to.be.true
       expect(
         postRequestStub.getCall(1).calledWith(PROD_URL_BACKEND, '', '/submit-event', {
           identityId: TEST_IDENTITY,
-          event: REFERRER_EVENT,
+          event: FIRST_PAGE_VISIT,
           attributes: {
+            url,
             referrer: 'https://arcx.money/',
+            utm: {
+              source: 'google',
+              medium: 'cpc',
+              campaign: 'brand',
+            },
           },
         }),
       ).to.be.true
 
       globalJsdom()
     })
 
-    it('makes a PAGE call if trackPages is true', async () => {
+    it('does not make a FIRST_PAGE_VISIT call if trackPages, referrer and UTM configs are set to false', async () => {
```

Comment:
add test where we have single config's tracking option set to true, and 2 are set to falce